<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frechet Distance Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>400</height>
              </size>
             </property>
             <property name="title">
              <string>Edit Graphs</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QPushButton" name="btn_change_graph">
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Change Graph</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_deleteGraph">
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Delete Graph</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lb_graph_red">
                  <property name="maximumSize">
                   <size>
                    <width>215</width>
                    <height>10</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Graph 1 - Red one</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_graph_red">
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="maximumSize">
                   <size>
                    <width>215</width>
                    <height>10</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Graph 2 - Blue one</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_graph_blue">
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btn_startCalc">
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Start calculating Frech√©t Distance</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>400</height>
              </size>
             </property>
             <property name="title">
              <string>Additional Parameters</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
